src/App.jsx

```diff
@@ -9,32 +10,19 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0]);
```

Comment:
variable name indicates that it's just id, but you set tab object

Reply:
But i'm using exact Id from object, not the object itself:)